railties/lib/rails/generators/rails/app/templates/config/initializers/content_security_policy.rb.tt

```diff
@@ -20,6 +20,9 @@
 #   config.content_security_policy_nonce_generator = ->(request) { request.session.id.to_s }
 #   config.content_security_policy_nonce_directives = %w(script-src style-src)
 #
+#   # Automatically add `nonce` to permitted directives.
+#   # config.content_security_policy_nonce_auto = true
```

Comment:
Is there a reason you'd want this to false? Because I'm thinking it's yet another extra config that I'd rather avoid.

Reply:
To be honest, I was trying to stay as retro-compatible as possible.

Now, if we put that aside, I believe there could be applications in which you actually want to "choose" what scripts are allowed to run (which is the current behavior).

What do you think?

_Note_: Personally, I think I'd always have that set to `true` in my projects, but I didn't want to enforce it for everyone by default.

Reply:
Ok, let me rephrase. Is there a case where you have a CSP nonce directive but wouldn't want to set the nonce on a script or style tag?

Reply:
I don't have that much thoughts, it's been a while since I worked on frontend much, so I don't have a lot of experience with CSP. I might defer to someone else.